src/test/java/SeleniumHW/Seleniumhw2.java

```diff
@@ -1,10 +1,114 @@
 package SeleniumHW;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.ie.InternetExplorerDriver;
+import Helper.MIsc;
+import Web.MyDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import javax.annotation.Signed;
+
 public class Seleniumhw2 {
+    /**
+     * Testcase-1:
+     *
+     *
+     * Verify "Keep me signed in" is NOT selected for messenger login page
+     *
+     * Click "Log in" button
+     *
+     * Verify "Incorrect email or phone number" is displayed
+     *
+     * Verify "Continue" button is enabled
+     *
+     *
+     * @return
+     */
+    @Test
+
+    public void verifyKeepmeSignedIn() {
+        MyDriver.launchUrlOnNewWindow("https://facebook.com");
+
+        String messengerText = "Messenger";
+        By messengerLocator = By.linkText(messengerText);
+        WebElement messengerLink = MyDriver.getDriver().findElement(messengerLocator);
+
+        String hrefValue = messengerLink.getAttribute("href");
+        Assert.assertTrue(hrefValue.contains("messenger.com"));
+
+        messengerLink.click();
+        MIsc.pause(5);
+        String urlAfterLink = MyDriver.getDriver().getCurrentUrl();
+        Assert.assertTrue(urlAfterLink.contains("messenger.com"), "Url is incorrect");
+
+
+        By SignedinBoxLocator = By.xpath("//label[contains(text(),'Keep me signed in')]");
+        WebElement SignedInBox = MyDriver.getDriver().findElement(By.xpath("//input[@id='Keep me signed in]"));
+        Assert.assertTrue(SignedInBox.isSelected());
```

Comment:
We are expecting "Keep me signed in" is NOT selected, so you supposed to do assertFalse() in above line.